test/integration/searchRanking.js

```diff
@@ -17,200 +19,198 @@ function makeSearchRequestAndCheckExpectations(search, assertions) {
     .end(assertions);
 }
 
-function rankTop3Results(res, className) {
+function rankTopResults(res, className) {
   const $ = cheerio.load(res.text);
   const searchResults = [];
-  for (let i = 0; i < 3; i++) {
+
+  for (let i = 0; i < RESULTS_THRESHOLD; i++) {
```

Comment:
`RESULTS_THRESHOLD` needs adding as a parameter as has been done with the other functions please.